packages/react/src/components/header/components/headerActionBar/HeaderActionBarNavigationMenu.test.tsx

```diff
@@ -373,23 +372,6 @@ describe('<HeaderActionBarNavigationMenu /> spec', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
-  it('should not have basic accessibility issues when menu is open', async () => {
```

Comment:
Commit history says "Comment out broken tests". These should be returned and fixed to check accessibility errors?

Reply:
fixed